<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="CTLT095" ignored="0" name="Book Flight for 2 Passengers" tests="5" failures="4" timestamp="15 Apr 2020 12:23:55 GMT" time="21.151" errors="0">
  <testcase name="open_Registration" time="5.896" classname="testcases.SanityTest"/>
  <testcase name="user_Registration" time="8.906" classname="testcases.SanityTest">
    <failure type="exception.MyException" message="Failed To Click On:By.name: register">
      <![CDATA[exception.MyException: Failed To Click On:By.name: register
at base.BasePage.clickOn(BasePage.java:255)
at pages.RegistrationPage.confirmRegistration(RegistrationPage.java:40)
at testcases.SanityTest.user_Registration(SanityTest.java:39)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- user_Registration -->
  <testcase name="confirm_User_Registration" time="0.001" classname="testcases.SanityTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at testcases.SanityTest.confirm_User_Registration(SanityTest.java:44)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- confirm_User_Registration -->
  <testcase name="flight_Reservation" time="0.001" classname="testcases.SanityTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at testcases.SanityTest.flight_Reservation(SanityTest.java:50)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- flight_Reservation -->
  <testcase name="flight_Confirmation" time="0.001" classname="testcases.SanityTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at testcases.SanityTest.flight_Confirmation(SanityTest.java:58)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- flight_Confirmation -->
</testsuite> <!-- Book Flight for 2 Passengers -->
